og.module

```diff
@@ -248,6 +248,9 @@ function og_entity_create_access(AccountInterface $account, array $context, $bun
     $handler = \Drupal::service('plugin.manager.entity_reference_selection');
     $handler = $handler->getInstance($options);
 
+    // @todo: remove deprecated call to Og::getSelectionHandler.
```

Comment:
FYI `@todo` docblock comments shouldn't have a colon and should start with a capital letter.

Reply:
Hi @damienmckenna, thanks for the input.

PS: did you know you can provide code suggestions on Github with the ± icon in the editor?